tests/schemas/api_v3/transaction.schema.js

```diff
@@ -30,6 +30,17 @@ const block = {
 	height: Joi.number().integer().min(1).required(),
 	timestamp: Joi.number().integer().positive().max(getCurrentTime())
 		.required(),
+	isFinal: Joi.boolean().required(),
+};
+
+const transactionMetaReceipientSchema = {
+	address: Joi.string().pattern(regex.ADDRESS_LISK32).required(),
+	publicKey: Joi.string().pattern(regex.PUBLIC_KEY).allow(null).required(),
+	name: Joi.string().pattern(regex.NAME).allow(null).required(),
```

Comment:
I think these two properties in `transactionMetaReceipientSchema` should be optional. Please re-verify.

Reply:
According to the tech design, meta itself is optional. But these are not.

```
      "meta": { // Optional
        "recipient": {
          "address": "lsk24cd35u4jdq8szo3pnsqe5dsxwrnazyqqqg5eu",
          "publicKey": "2ca9a7...c23079",
          "name": "genesis_49",
        },
      },
``